### PR TITLE
Eletrolux alert refinement

### PR DESCRIPTION
**Pull request by kingy444**
- Default to only send Alert Notications for severity of `ALERT`
- Ignore alerts with status `NOT_NEEDED`
- `WARNING` and `DIAGNOSTIC` default as off.
- Other states will still fall into `ALERT` status until they are defined

### CODE REVIEW

File: custom_components/electrolux_status/translations/de.json [hunk 1]
```diff
@@ -27,7 +29,9 @@
             "user": {
                 "data": {
                     "language": "Sprache",
-                    "notifications": "Senden Sie Benachrichtigungen für erhöhte Warnungen",
+                    "notifications": "Erhöhen Sie Benachrichtigungen für Benachrichtigungen auf Warnstufe",
+                    "notifications_diagnostic": "Erhöhen Sie Benachrichtigungen für Diagnosebereich Hinweise",
```
Comment (dschido) [inline custom_components/electrolux_status/translations/de.json:33]: Senden Sie Benachrichtigungen für Alarmmeldungen

**Comment by dschido** [comment]
Wouldn't it be better to consider the status (if possible) as well? If status is “NOT_NEEDED” then ignore it?
You can see from the examples in #87  that it is always this status, with varying severity

**Comment by dschido** [comment]
could you please adjust the german translation?

"notifications": "Benachrichtigungen für Alarme versenden",
"notifications_diagnostic": "Benachrichtigungen für Diagnosemeldungen versenden",
"notifications_warning": "Benachrichtigungen für Warnungen versenden",

**Comment by kingy444** [comment]
> could you please adjust the german translation?
> 
> 
> 
> "notifications": "Benachrichtigungen für Alarme versenden",
> 
> "notifications_diagnostic": "Benachrichtigungen für Diagnosemeldungen versenden",
> 
> "notifications_warning": "Benachrichtigungen für Warnungen versenden",

These are just generated by google translate (translate.py). 
I won't be manually changing any of them as this will just get overwritten in the future.

If having these is an issue I suggest removing the translations completely

**Comment by albaintor** [comment]
Thank you @kingy444

**Pull request merged**